frontend/public/components/pod.jsx

```diff
@@ -200,24 +201,33 @@ const Details = ({obj: pod}) => {
             <dd>{podPhase(pod)}</dd>
             <dt>Restart Policy</dt>
             <dd>{getRestartPolicyLabel(pod)}</dd>
-            {activeDeadlineSeconds && <React.Fragment>
-              <dt>Active Deadline</dt>
-              {/* Convert to ms for formatDuration */}
-              <dd>{formatDuration(activeDeadlineSeconds * 1000)}</dd>
-            </React.Fragment>}
+            {
+              activeDeadlineSeconds &&
+                <React.Fragment>
+                  <dt>Active Deadline</dt>
+                  {/* Convert to ms for formatDuration */}
+                  <dd>{formatDuration(activeDeadlineSeconds * 1000)}</dd>
+                </React.Fragment>
+            }
             <dt>Pod IP</dt>
             <dd>{pod.status.podIP || '-'}</dd>
             <dt>Node</dt>
             <dd><NodeLink name={pod.spec.nodeName} /></dd>
+            {
+              securityScan &&
+                <React.Fragment>
+                  <dt>Security Scan</dt>
+                  <dd>`${securityScan} fixable packages`</dd>
```

Comment:
This should work, right?

```jsx
<dd>{securityScan} fixable packages</dd>
```

Reply:
Ha, yeah. Totally a copy paste oversight